src/lib/components/shell/Header.svelte

```diff
@@ -4,8 +4,12 @@
   import ShowHide from '$svelteui/functions/ShowHide.svelte';
   import Button from '$svelteui/ui/Button.svelte';
   import { firebaseConfig } from '$sveltefire/config';
+  let screenWidth: number;
+  const mobileView = 639;
```

Comment:
I'm very doubtful about this. I think you already have one way to differentiate this berakpoint.

Reply:
Good use of Svelte's window methods, but yes, Tailwind has some easy options for us here. Use `hidden lg:block` on one you want to show on desktop and then `lg:hidden` on the one that needs to hide at that breakpoint.